content/docs/studio/view-settings.md

```diff
@@ -0,0 +1,100 @@
+# Additional Settings for a View
+
+If you are creating a view for a DVC repo, and if the DVC repo is at the root of
+the Git repository and does not reference remote/cloud storage, then you can
+successfully visualize it without specifying additional settings.
+
+Alternatively, you could create views from:
+
+- Non-DVC repositories
+- Project sub-directories in a monorepo
+- Custom files in your repository or remote/cloud storage
+
+In each of these scenarios, you will need to specify advanced settings for DVC
+Studio to be able to access the data required for visualization. Details are
+given below.
+
+## Non-DVC repositories
+
+DVC Studio creates views by identifying datasets, metrics and hyperparameters
+defined in your Git repositories. These details (datasets, metrics and
+hyperparameters) are stored in your Git repositories as CSV, JSON or YAML files.
+You can add these details to your Git repositories in two ways:
+
+1. **Set up DVC repositories**: You can use [DVC](https://dvc.org/) and Git to
```

Comment:
DVC is not only about not bloating Git repo. It's about pipelines, connecting then with CML, etc. We should be explaining these benefits better I think

Reply:
We should have a separate section/paragraph on DVC and its benefits, and link it to the DVC doc. For now, I'm adding this to the current paragraph: "DVC also
   enables you to
   [share your data and model files](/doc/use-cases/sharing-data-and-model-files),
   [create data registries](/doc/use-cases/data-registries),
   [create data pipelines](/doc/start/data-pipelines), connect them with
   [CML](/doc/cml) for CI/CD in machine learning, and so on. Find more about the
   features and benefits of DVC [here](/doc/start)."

Reply:
ideally we should zoom out and summarize high level benefits here (e.g. pipelines == reproducibility).

One way can be - DVC goes beyond that. It lets you organize your project in a reproducible way, it gives you way to save your experiments holistically as Git commits - data, models, metrics, params ... You could use it to organize manage your data using Git flow (data registries) ... 

something like this ...